tsl/src/continuous_aggs/create.c

```diff
@@ -1751,44 +1751,59 @@ cagg_create(ViewStmt *stmt, Query *panquery, CAggTimebucketInfo *origquery_ht,
 	return;
 }
 
-/* entry point for creating continuous aggregate view
- * step 1 : validate query
- * step 2: create underlying tables and views
- */
-DDLResult
-tsl_process_continuous_agg_viewstmt(ViewStmt *stmt, const char *query_string, void *pstmt,
-									WithClauseResult *with_clause_options)
+static bool
```

Comment:
Mismatching type of return value: bool here, but returns DDL_DONE. I would probably make this return void since there is only one possible return value.